src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/Diagnostic/Test/Verifiers/CSAnalyzerVerifier`1.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+
+namespace $safeprojectname$
+{
+    public static partial class CSharpAnalyzerVerifier<TAnalyzer>
```

Comment:
Why have these as source code in the templates instead of adding them the the library itself?

Reply:
The ones in the library cannot be edited as part of the project.